firewood/src/merkle/node.rs

```diff
@@ -229,7 +244,7 @@ impl Node {
                 is_encoded_longer_than_hash_len: OnceLock::new(),
                 inner: NodeType::Branch(
                     BranchNode {
-                        // path: vec![].into(),
+                        path: vec![].into(),
```

Comment:
🤔  I have to see where this is being called. The branch-node size is dynamic now so this function doesn't really make sense.

Reply:
I guess the value could have always grown too, so the behaviour is consistent